assets/components/signout/signout.jsx

```diff
@@ -25,22 +25,34 @@ function buildUrl(returnUrl: ?string): string {
 
 }
 
+function Signout(props: PropTypes) {
+  if (props.isSignedIn) {
+    return (
+      <a className="component-signout" href={buildUrl(props.returnUrl)}>
+        Not you? Sign out
+      </a>
+    );
+  }
+}
 
-// ----- Component ----- //
+// ----- Map State/Props ----- //
 
-export default function Signout(props: PropTypes) {
+function mapStateToProps(state) {
 
-  return (
-    <a className="component-signout" href={buildUrl(props.returnUrl)}>
-      Not you? Sign out
-    </a>
-  );
+  return {
+    email: state.page.user.email,
+    isSignedIn: state.page.user.isSignedIn,
+  };
 
 }
+// ----- Component ----- //
```

Comment:
To be consistent with the rest of the files, I would put the comment:
// ------ Exports -----//
and also push it to the bottom of the file.